hippod/report_generator.py

```diff
@@ -132,15 +160,18 @@ def sanitize_description(description_path):
                         output_text.write(line)
 
 
-        def check_image_reference(description_path, attach_path):
+        def check_image_reference(description_path, attach_path, data_type):
```

Comment:
try to comment what this function do, in a high level way. Reading the regex is not so easy, the function name "check_image_reference()" seems to return a bool but it does not only "check". Rather the name of the method should be adjusted and/or a comment is missing.